include/exiv2/slice.hpp

```diff
@@ -114,7 +69,7 @@ namespace Exiv2
              * Throw an exception when index is too large.
              *
              * @throw std::out_of_range when `index` will access an element
-             * outside of the slice
+             *     outside of the slice
```

Comment:
[suggestion] We could write this line in one single line since it will not pass the limit of characters we have configured in our .clang-format file:

```
ColumnLimit: 120
```

[discussion] What do you think about using the doxygen comments in this way: 

```cpp
// Current style

            /*!
             * Throw an exception when index is too large.
             *
             * @throw std::out_of_range when `index` will access an element
             *     outside of the slice
             */
            inline void rangeCheck(size_t index) const

// proposed style

            /// Throw an exception when index is too large.
            /// @throw std::out_of_range when `index` will access an element outside of the slice
            inline void rangeCheck(size_t index) const
```
With the current Doxygen configuration, the `\brief` is implicitly added in the first line. As you could see, this Doxygen style reduce considerably the size of headers and increase the code readability (IMHO).